modules/virt-must-gather-options.adoc

```diff
@@ -33,67 +33,73 @@ Using too many parallel processes can cause performance issues. Increasing the m
 
-`gather_images`:: Collect image and image stream custom resource information. This script is only compatible with the `PROS` variable.
+`/usr/bin/gather --images`:: Collect image, image-stream-tags, and image stream custom resource information. This script is compatible only with the `PROS` variable.
 
 [id="usage-and-examples_{context}"]
 == Usage and examples
 
 Environment variables are optional. You can run a script by itself or with one or more compatible environment variables.
 
+To customize the data that `must-gather` collects, you append a double dash (`--`) to the command, followed by a space and one or more compatible parameters.
+
 [cols="1a,1a", options="header"]
 .Compatible parameters
 |===
 |Script |Compatible environment variable
-|`gather_vms_details`
+|`/usr/bin/gather`   |* `PROS=<number_of_processes>`
+|`/usr/bin/gather --vms_details`
 |* For a namespace: `NS=<namespace_name>`
 
 * For a VM: `VM=<vm_name> NS=<namespace_name>`
 
 * `PROS=<number_of_processes>`
 
-|`gather`   |* `PROS=<number_of_processes>`
-|`gather_images`   |* `PROS=<number_of_processes>`
+
+|`/usr/bin/gather --images`   |* `PROS=<number_of_processes>`
 |===
 
-To customize the data that `must-gather` collects, you append a double dash (`--`) to the command, followed by a space and one or more compatible parameters.
+
 
 .Syntax
 
+
+
 [source,terminal,subs="attributes+"]
 ----
 $ oc adm must-gather \
   --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion} \
   -- <environment_variable_1> <environment_variable_2> <script_name>
 ----
 
-.Detailed VM information
+.Default data collection limited to three parallel processes
 
-The following command collects detailed VM information for the `my-vm` VM in the `mynamespace` namespace:
+The following command collects default `must-gather` information by using a maximum of three parallel processes:
 
 [source,terminal,subs="attributes+"]
 ----
 $ oc adm must-gather \
   --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion} \
-  -- NS=mynamespace VM=my-vm gather_vms_details <1>
+  -- PROS=3 /usr/bin/gather
 ----
-<1> The `NS` environment variable is mandatory if you use the `VM` environment variable.
 
-.Default data collection limited to three parallel processes
+.Detailed VM information
 
-The following command collects default `must-gather` information by using a maximum of three parallel processes:
+The following command collects detailed VM information for the `my-vm` VM in the `mynamespace` namespace:
 
 [source,terminal,subs="attributes+"]
 ----
 $ oc adm must-gather \
   --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion} \
-  -- PROS=3 gather
+  -- NS=mynamespace VM=my-vm /usr/bin/gather --vms_details <1>
 ----
+<1> The `NS` environment variable is mandatory if you use the `VM` environment variable.
+
 
 .Image and image stream information
```

Comment:
image, image-stream, and image-stream-tags: https://github.com/kubevirt/must-gather#targeted-gathering---images-information